lib/src/main/kotlin/com/lemonappdev/konsist/core/scope/KoScopeCreatorImpl.kt

```diff
@@ -97,31 +97,39 @@ internal class KoScopeCreatorImpl : KoScopeCreator {
         return KoScopeImpl(koFiles)
     }
 
-    override fun scopeFromDirectory(path: String): KoScope {
-        val directory = File(path)
-        require(directory.exists()) { "Directory does not exist: $path" }
-        require(!directory.isFile) { "Path is a file, but should be a directory: $path" }
+    override fun scopeFromDirectory(path: String, resolvePathFromProjectRoot: Boolean): KoScope {
```

Comment:
Let's change this API a bit.
Rename 2nd argument to `absolutePath` (default false).
Let's mention in the kdocs docs that by default method resolves path from project root.

Sampe for `scopeFromFile` method